distribution/lib/Standard/Base/0.0.0-dev/src/Network/Email/Email_Provider_SPI.enso

```diff
@@ -0,0 +1,38 @@
+import project.Data.Vector.Vector
+import project.Function.Function
+import project.Meta
+import project.Metadata.Display
+import project.Metadata.Widget
+from project.Metadata.Widget import Single_Choice
+from project.Nothing import all
+
+## ---
+   private: true
+   ---
+   Internal class for enabling SPI for Email_Provider types.
+type Email_Provider_SPI
+    private Value typ
```

Comment:
- there are various SPI providers around the code base
    - some of them were written from scratch
    - some of them were adjusted from previous "service loader" code
    - you decided to copy example based on "server loader" - e.g. it is far from optimal
- this be done in a better way

Reply:
What does the "a" indicate in the blueprint? Is there an SPI in the code base that would be a better example to copy?